openapi/openapi.yaml

```diff
@@ -1080,6 +1080,35 @@ paths:
               schema:
                 $ref: "#/components/schemas/PythonPackageMetadataResponseError"
 
+  /python/get-package-from-imports:
+    get:
+      tags: [PythonPackages]
+      x-openapi-router-controller: thoth.user_api.api_v1
+      operationId: get_package_from_imported_packages
+      summary: List imported packages' correct names.
+      parameters:
+        - name: import_name
+          in: query
+          required: true
+          description: >
+            Names of the Python Packages for the given import package name.
+          schema:
+            type: string
+            default: "tensorflow"
```

Comment:
Might be a good idea to show the sklearn/scikit-learn example. Ex. `sklearn.cluster.MiniBatchKMeans` on the input and `scikit-learn` in the response example (or any other package where package name does not correspon with the import).